__tests__/articles_comments-endpoint.test.js

```diff
@@ -0,0 +1,44 @@
+const data = require('../db//data/test-data')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const db = require('../db/connection')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('Testing Articles Comments Endpoint', () => {
+    describe('/api/articles/:articles_id/comments', () => {
+        test('200: get 200 response', () => {
+            return request(app).get('/api/articles/1/comments').expect(200)
+        })
+        test('200: returns an array of article comments', () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then(({body}) => {
+                body.forEach(comment => {
+                    expect(typeof comment.comment_id).toBe('number')
+                    expect(typeof comment.body).toBe('string')
+                    expect(typeof comment.article_id).toBe('number')
+                    expect(typeof comment.author).toBe('string')
+                    expect(typeof comment.votes).toBe('number')
+                    expect(typeof comment.created_at).toBe('string')
+                })
+            })
```

Comment:
This is a good start, but body should be an object with an appropriate key which holds the array of comments.

We also should be asserting exactly what the article_id is for each comment here as this is our way of checking if the comments that are coming back are actually the ones we want.

We are receiving an array here as well, which means we should be asserting the length of the array before this forEach. To ensure the array contains some elements, because if it didnt this test would still pass which is a false positive.